src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -190,10 +190,10 @@ trait ScalaSettings extends AbsScalaSettings
   val nopredef        = BooleanSetting    ("-Yno-predef", "Compile without importing Predef.")
     .withPostSetHook(postSetExclusive(Ysysdef, Ypredef))
   val Ysysdef: StringSetting =
-                        StringSetting     ("-Ysysdef", "imports", "Supply a list of packages/objects to be imported into the global scope as system imports", "java.lang._, scala._")
+                        StringSetting     ("-Ysysdef", "imports", "Supply a comma separated list of fully qualified packages/objects to be imported into the global scope as system imports. Use an empty string or the string \"_\" to configure for no system imports.", "java.lang._, scala._")
```

Comment:
Now it's a long line, most other `-Y` descriptions fit on a single terminal line.. Something like:

"Comma-separated list of fully qualified system imports, added to every source file. `_` for none."

"Comma-separated list of fully qualified Predef imports, added if not clashing with top-level imports. `_` for none."